unified_planning/model/agent.py

```diff
@@ -0,0 +1,61 @@
+# Copyright 2021 AIPlan4EU project
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import unified_planning
+from unified_planning.shortcuts import *
+import unified_planning.model.operators as op
+from unified_planning.exceptions import (
+    UPProblemDefinitionError,
+    UPTypeError,
+    UPValueError,
+    UPExpressionDefinitionError,
+)
+from unified_planning.model.walkers import OperatorsExtractor
+from fractions import Fraction
+from typing import List, Dict, Set, Union, Optional
+from unified_planning.model.expression import ConstantExpression
+from unified_planning.model.abstract_problem import AbstractProblem
+from unified_planning.model.mixins import (
+    ActionsSetMixin,
+    FluentsSetMixin,
+)
+
+
+class Agent(
+    FluentsSetMixin,
+    ActionsSetMixin,
+):
+
+    """Represents an Agent."""
+
+    def __init__(
+        self,
+        name: str,
+        env: "unified_planning.environment.Environment" = None,
+        ma_problem: "unified_planning.model.MultiAgentProblem" = None,
```

Comment:
This is not an optional parameter
```suggestion
        ma_problem: "unified_planning.model.MultiAgentProblem",
```